static/scss/generic/_modal.scss

```diff
@@ -221,7 +225,7 @@
     border: solid transparent; /* Using border instead of padding to keep bg image in place */
     overflow: hidden;
 }
-.vbox-prev{
+/*.vbox-prev{
```

Comment:
Only reason why I have commented it out similar to the iOS code is incase this is required. It's an external batch of CSS from venobox would thought it pertinent to keep it.